### ISSUE TITLE
Training with longer examples leads to `InvalidArgumentError`

### ISSUE THREAD

**Issue by pschwllr** [as-ideas/headliner#17]
When I try to train a `SummarizerTransformer` on longer training examples I get the following error: `InvalidArgumentError:  Incompatible shapes: [1,11,64] vs. [1,8,64]` in `train_step`. It looks like it is depending on the length of the targets.

Minimal example:
```
from headliner.trainer import Trainer
from headliner.model.summarizer_transformer import SummarizerTransformer

data = [
        ('You are the stars, earth and sky for me!', 'I love you I love you I love you.'),
        ('You are the stars, earth and sky for me!', 'I love you.')
]

summarizer = SummarizerTransformer(embedding_size=64, max_prediction_len=20)
trainer = Trainer(batch_size=1, steps_per_epoch=100)
trainer.train(summarizer, data, num_epochs=1)
```

Leads to an `InvalidArgumentError` while

```
from headliner.trainer import Trainer
from headliner.model.summarizer_transformer import SummarizerTransformer

data = [
        ('You are the stars, earth and sky for me!', 'I love you.'),
        ('You are the stars, earth and sky for me!', 'I love you.')
]

summarizer = SummarizerTransformer(embedding_size=64, max_prediction_len=20)
trainer = Trainer(batch_size=1, steps_per_epoch=100)
trainer.train(summarizer, data, num_epochs=1)
```
without the ` ('You are the stars, earth and sky for me!', 'I love you I love you I love you.')` pair, works fine. 

I use:
```
python==3.6
tensorflow==2.0.0
headliner== 0.0.22
```
It does not depend on if I run it on a gpu or cpu only. 

Can you reproduce this bug?

**Comment by cschaefer26** [comment]
Hi, thx for the minimal example, I'll look into it right now.

**Comment by cschaefer26** [comment]
I could reproduce the issue and fix it - there was a problem with the positional encoding mask in case of small vocab size and large sequence lengths. Fixed in 0.0.23. Thanks for reporting!
